book/scheduling.md

```diff
@@ -489,7 +484,10 @@ class JSContext:
 
 Note that in the asynchronous case, the `XMLHttpRequest_send` method starts a
 thread and then immediately returns. That thread will run in parallel
-with the browser's main work until the request is done.
+with the browser's main work until the request is done.^[In theory two
+parallel requests could race while accessing the cookie jar; I'm not
+fixing this out of expediency but a proper implementation would have
+locks for the cookie jar.]
```

Comment:
I think this is the best we can really do at the moment. Hash operations might be atomic in Python but I didn't feel confident enough in this fact to put it in the footnote.

Reply:
not critical to fix this in the printed book.